### PR TITLE
Update prevent-changes-to-security-settings-with-tamper-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
Added licenses which includes Intune

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7181

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ If you are part of your organization's security team, and your subscription incl
 
 - You must have appropriate [permissions](/microsoft-365/security/defender-endpoint/assign-portal-access) assigned, such as global admin, security admin, or security operations.
 
-- Your organization uses [Intune to manage devices](/intune/fundamentals/what-is-device-management). ([Intune licenses](/intune/fundamentals/licenses) are required; Intune is included in Microsoft 365 E5.)
+- Your organization uses [Intune to manage devices](/intune/fundamentals/what-is-device-management). ([Intune licenses](/intune/fundamentals/licenses) are required; Intune is included in Microsoft 365 E5/E3, Enterprise Mobility + Security E5/E3, Microsoft 365 Business Premium, Microsoft 365 F1/F3, Microsoft 365 Government G5/G3 and Intune for Education.)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md]:
```suggestion
- Your organization uses [Intune to manage devices](/intune/fundamentals/what-is-device-management). ([Intune licenses](/intune/fundamentals/licenses) are required; Intune is included in Microsoft 365 E5/E3, Enterprise Mobility + Security E5/E3, Microsoft 365 Business Premium, Microsoft 365 F1/F3, Microsoft 365 Government G5/G3, and Intune for Education.)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**